runtime/src/main/java/com/dylibso/chicory/runtime/InterpreterMachine.java

```diff
@@ -1787,13 +1786,6 @@ private static void TABLE_GET(MStack stack, Instance instance, long[] operands)
 
     private static void GLOBAL_SET(MStack stack, Instance instance, long[] operands) {
         var id = (int) operands[0];
-        var mutabilityType =
```

Comment:
- note to self - At the end of this process, we should review the Machine to remove any other redundant check.

Reply:
Yes! When writing the AOT, I noticed various checks in the interpreter that seemed like they should have been handled during validation. I should have kept notes.

Reply:
Tracked in #422